tests/Aql/ExecutorTestHelper.h

```diff
@@ -222,7 +264,33 @@ struct ExecutorTestHelper {
     _expectedStats = stats;
     _testStats = true;
     return *this;
-  };
+  }
+
+  template <typename E>
+  auto setExecBlock(typename E::Infos infos) -> ExecutorTestHelper& {
+    // TODO: this unique_ptr goes out of scope and is free'd, but I think
```

Comment:
I think you can remove this todo.
The unique_ptr is handled in the _execNodes.
Then you just have a reference to the unique_pointer, which is not deleted here, but on destruction of _execNodes.

Important is that. testeeNode.get() still yields a valid pointer, which in my opinion it does here.

Reply:
Yes, the comment was written before _execNodes existed.